packages/deprecated/package.json

```diff
@@ -19,7 +19,8 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"dependencies": {
-		"@babel/runtime": "^7.0.0-beta.52"
+		"@babel/runtime": "^7.0.0-beta.52",
```

Comment:
We still need to run `npm install` to update `package-lock.json`.

Reply:
done